### PR TITLE
Week 3 Goals and Economics Presentation Outline

### PR DESCRIPTION
**Pull request by WesleyM314**
I rebased the repo so it doesn't include any of the deleted Rust code; these commits are just from merging in changes to management docs and the economics outline.

### CODE REVIEW

File: adv_topics/econ.md [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+		* Runescape Grand Exchange - fully emergent economy 
+* Topic 1 - A deep dive into emergent economies and what makes an economy emergent - 20 mins 
+	* Overview of research [paper](http://ianparberry.com/pubs/econ.pdf)
+	* Emergent Economy 
+	* Price beliefs in economy
+		* Bid
+		* Ask
+		* Sale
+		* Purchase
+* Topic 2 - An explanation on how we can go from a static economy into a fully emergent one - 30 mins
```
Comment (nfarnan) [inline adv_topics/econ.md:19]: So the overall presentation is only set for 45 minutes, not the full lecture time. This plus 15 minutes allowed for questions throughout will still not take up most of lecture, but this course requirement was set for 75-minute lectures, and I'm not having you guys do _more_ work over the summer. SO, keep Intro and Topic1 as-is. They look great for the first adv. topic discussion. Cut Topic 3 entirely (the application focus will be on your game, no need to get the sim running in MatLab and then re-implement in Rust). And convert Topic2 to a high-level overview or a look at only 1 of the bullet points there to fill up an addition 10min. Yes, this cuts out the bulk of the "meat" of your presentation, but you guys are the first group to go, so I'd be happy with a more introductory overall presentation. Also, I'm sure you'll be able to fill any free time this provides for you over the week with more work on game code!

File: management/Farming_manage.md [hunk 2]
```diff
@@ -9,22 +9,25 @@
...
 		1. Create game "credits" in team repo
 3. 06/16 - 06/22
-	* Manager: NAME
+	* Manager: Wesley Miller (wam34)
 	* Goals:
-		1. GOAL1
-		...
+		1. Design and implement sprite for the farmer (player character)
+		2. Implement movement of farmer from top-down view
+		3. Design and implement the tiling background of the ground and set size/boundaries of farm
```
Comment (nfarnan) [inline management/Farming_manage.md:16]: This is a bit ambitious as stated, maybe scale back to just be able to repaste a single tile to cover the entire camera view. If you get more than that, awesome! Part of the role of these checks is to make sure they both achievable and sufficient.

File: management/Farming_manage.md [hunk 3]
```diff
@@ -39,7 +42,7 @@
 		1. GOAL1
 		...
 9. 07/28 - 08/03
-	* Manager: REQUIRED
+	* Manager: Natasha Kamtekar (nak142)
```
Comment (nfarnan) [inline management/Farming_manage.md:45]: As a note Natasha should not be filling in as manager for all three of these weeks. That would not be fair. All of you will need to pick up the manager responsibilities as far as merging into the canonical repo from 7/14-7/27.

**Review by nfarnan — CHANGES_REQUESTED** [review]
Overall I need a few minor changes to scale somethings back to a more manageable size. Overall looks good on details, though. Make those changes and I'll approve.

In the future, however, please further split up 1 PR per contribution (e.g., different PRs for adv. topic outlines and management goals). This can be accomplished by using branches for different contributions

**Comment by nfarnan** [comment]
Also, way to go on getting the rust code out of the commit history!

**Review by WesleyM314 — COMMENTED** [review]
Made and pushed the requested changes, altering the goal to simply tiling a single tile and shortening the presentation outline.

**Review by nfarnan — APPROVED** [review]
All changes good!

**Pull request merged**